esp/esdllib/esdl_script.cpp

```diff
@@ -2883,6 +2883,119 @@ class CEsdlTransformOperationDelay : public CEsdlTransformOperationWithoutChildr
     }
 };
 
+class CEsdlTransformOperationUpdateMaskingContext : public CEsdlTransformOperationBase
+{
+    class Update : public CEsdlTransformOperationWithoutChildren
+    {
+    public: // IEsdlTransformOperation
+        virtual bool exec(CriticalSection* crit, IInterface* preparedForAsync, IEsdlScriptContext* scriptContext, IXpathContext* targetContext, IXpathContext* sourceContext) override
+        {
+            return true;
+        }
+    public:
+        virtual bool apply(IDataMaskingProfileContext& masker, IXpathContext& sourceContext) const = 0;
+        using CEsdlTransformOperationWithoutChildren::CEsdlTransformOperationWithoutChildren;
```

Comment:
Why was this using necessary?

Reply:
It's shorthand for giving the class a constructor that takes the same arguments as the base class and automatically calls the base class.